ADO.NET/AdoOauthSample/OAuthSampleApp.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Vertica.Data.VerticaClient;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using System.Configuration;
+
+internal class AdoOauthSample
+{
+    static void Main(string[] args)
+    {
+        VerticaConnectionStringBuilder vcsb = new VerticaConnectionStringBuilder();
+        vcsb.ConnectionString = ConfigurationManager.ConnectionStrings["ConnectionString"].ConnectionString;
+
+      //  vcsb.OAuthJsonConfig = GetTokenRefreshJsonConfig();
+        ConnectToDatabase(vcsb.ConnectionString);
+    }
+
+    public static void ConnectToDatabase(string connectionString)
+    {
+        try
+        {
+            using (VerticaConnection conn = new VerticaConnection(connectionString))
+            {
+                conn.Open();
+
+                using (VerticaCommand command = conn.CreateCommand())
+                {
+                    Console.WriteLine("Connection Successful");
+                    command.CommandText = "SELECT user_id, user_name FROM USERS ORDER BY user_id";

```

Comment:
The query is defined in the app.config file.   We should either remove that setting or use it here.   I think if we make it easier to modify the query, we should consider sharing the ResultSetPrinter between the OAuth and GettingStarted samples.   Is there a reasonably easy way to do that?

Reply:
I'll remove the query from the app.config and look into printing out the results nicely.